jsapp/js/components/common/audioPlayer.tsx

```diff
@@ -69,61 +85,73 @@ class AudioPlayer extends React.Component<AudioPlayerProps, AudioPlayerState> {
       this.audioInterface.currentTime === this.state.totalTime &&
       this.state.isPlaying
     ) {
-      this.onPlayStatusChange()
+      this.onPlayStatusChange();
     }
 
-    this.setState({currentTime: this.audioInterface.currentTime})
+    this.setState({currentTime: this.audioInterface.currentTime});
   }
 
   onPlayStatusChange() {
     if (!this.state.isPlaying) {
-      this.audioInterface.play()
+      this.audioInterface.play();
     } else {
-      this.audioInterface.pause()
+      this.audioInterface.pause();
     }
 
     this.setState({
       isPlaying: !this.state.isPlaying,
-    })
+    });
   }
 
   onSeekChange(newTime: string) {
-    this.audioInterface.currentTime = parseInt(newTime)
+    this.audioInterface.currentTime = parseInt(newTime);
 
     this.setState({
       currentTime: parseInt(newTime),
-    })
+    });
+  }
+
+  renderPlayer() {
+    return (
+      <React.Fragment>
+        <bem.AudioPlayer__controls>
+          <Button
+            type='bare'
+            startIcon={this.state.isPlaying ? 'pause' : 'caret-right'}
+            size='l'
+            color='blue'
+            onClick={this.onPlayStatusChange.bind(this)}
```

Comment:
Do you have any established convention on naming those `data-cy="?"`? Here I think it would be good to make sure it has component name in it, e.g. `data-cy="AudioPlayer pauseplay"`. But maybe you already use `audio-player-pauseplay` or `audioplayer_pauseplay` convention in your tests?

Reply:
I've seen on https://github.com/wetbadger/kpi/blob/cypress/cypress/cypress that you're using `data-cy="delete"` and similar short names. I think it would be wise at this point to include more context in those. It is quite possible that at some point given view would have multiple delete options visible at the same time, so "delete" could lead to issues. Maybe it could be "form row buttons", "form row more actions", "more actions delete", "delete confirm modal checkbox", "delete confirm modal delete"? :)

Reply:
So for this we can go with "audio player pauseplay" :)

Reply:
"So lowercase words separated with space character" could go somewhere in cypress test configuration as comment?